src/Ojs/CoreBundle/Service/Mailer.php

```diff
@@ -164,6 +193,15 @@ public function getJournalStaff()
         return $this->em->getRepository(User::class)->findUsersByJournalRole($roles);
     }
 
+    /**
+     * @param Journal $journal
+     * @return SubscribeMailList[]
```

Comment:
Method returns an ArrayCollection. You can either convert that to an array using its `toArray` method or correct this PHPDoc block.